core/src/SplitView.html

```diff
@@ -0,0 +1,320 @@
+<svelte:window on:resize="onResize()"/>
+<div class="lui-split-view">
+  {#if collapsed}
+  <div class="splitViewSeparator isCollapsed">
+    <a class="lui-collapse-btn" on:click:stopPropagation="expand(event)">
+      <span class="sap-icon--collapse-group sap-icon--s"></span>
+    </a>
+  </div>
+  <h1 class="fd-splitView__title">{splitViewSettings.title}</h1>
+  {:else}
+  <div id="splitViewDragger" on:dragSplitViewBorder="dragged(event)">
+    <div class="splitViewSeparator"></div>
+    <a class="lui-collapse-btn" on:click:stopPropagation="collapse(event)">
+      <span class="sap-icon--expand-group sap-icon--s"></span>
+    </a>
+  </div>
+  <div id="splitViewDraggerBackdrop"></div>
+  <div class="iframeSplitViewCnt"></div>
+  {/if}
+</div>
+
+<script>
+  import { LuigiElements } from './core-api';
+  import { Iframe, SplitViewSvc } from './services';
+  import {
+    GenericHelpers,
+    IframeHelpers,
+    RoutingHelpers
+  } from './utilities/helpers';
+
+  let rightContentHeight,
```

Comment:
Maybe it is a good idea to wrap all these variables under an object

Reply:
done

Reply:
done